### ISSUE TITLE
Realtime APIs use the wrong endpoint and 404s unless API base path is set to the realtime endpoint.

### ISSUE THREAD

**Issue by caiges** [fastly/fastly-js#18]
**Version**

`3.1.0`

**What happened**

Using the realtime APIs results in 404s since the endpoint is `rt.fastly.com` per the [documentation](https://developer.fastly.com/reference/api/metrics-stats/realtime/).

A workaround is to initialize a standalone API client and set its base path [accordingly](https://github.com/newrelic/fastly-to-insights/pull/4/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R19-R22):

```javascript
const fastlyApi = new ApiClient();
fastlyApi.basePath = "https://rt.fastly.com";
fastlyApi.authenticate(FASTLY_KEY);
const realtimeApi = new RealtimeApi(fastlyApi);
```

**Comment by Integralist** [comment]
👋🏻 Thanks for opening this issue @caiges I've opened an internal ticket to track this bug.

**Comment by Integralist** [comment]
I'm going to close this issue as I believe this issue was resolved in the 4.2.0 release back in June:
https://github.com/fastly/fastly-js/blob/main/CHANGELOG.md#v420-2023-06-20
